index.html

```diff
@@ -129,55 +129,412 @@ <h2 data-dft-for="">
+        <li>The installation process can occur through a <dfn>site-triggered
+        install prompt</dfn>: the site can programmatically request that the
+        user agent present an install prompt to the user. The user agent MAY
+        restrict the availability of this feature to cases where, for instance,
+        there are sufficient <a data-cite="appmanifest#installability-signals">
+          installability signals</a> to warrant <a data-cite=
+          "appmanifest#dfn-installed">installation</a> of the web application.
         </li>
       </ul>
+      <p>
+        In any case, the user agent MUST NOT <a>present an install prompt</a>
+        if the document is not <a data-cite=
+        "appmanifest#dfn-is-installable">installable</a>.
+      </p>
+      <p>
+        The user agent MAY, at any time (only if the document is <a data-cite=
+        "appmanifest#dfn-is-installable">installable</a>), run the <a>steps to
+        notify that an install prompt is available</a> at any time, giving the
+        site the opportunity to show a <a>site-triggered install prompt</a>
+        without the user needing to interact with the user agent UI.
+      </p>
+      <p>
+        To <dfn data-lt=
+        "presenting an install prompt|presentation of the install prompt">present
+        an install prompt</dfn>:
+      </p>
+      <ol>
+        <li>Show some user-agent-specific UI, asking the user whether to
+        proceed with installing the app. See <a data-cite=
+        "appmanifest#installation-sec">privacy and security considerations</a>
+        for recommendations relating to this UI. The <var>result</var> of this
+        choice is either {{AppBannerPromptOutcome|accepted}}
+          <a data-link-for="AppBannerPromptOutcome">accepted</a> or
+          <a data-link-for="AppBannerPromptOutcome">dismissed</a>.
+        </li>
+        <li>Return <var>result</var>, and <a>in parallel</a>:
+          <ol>
+            <li>If <var>result</var> is <a data-link-for=
+            "AppBannerPromptOutcome">accepted</a>, run the <a>steps to install
+            the web application</a>.
+            </li>
+          </ol>
+        </li>
+      </ol>
+      <p>
+        The <dfn>steps to notify that an install prompt is available</dfn> are
+        given by the following algorithm:
+      </p>
+      <ol>
+        <li>Wait until the {{Document}} of the <a>top-level browsing
+        context</a> is <a>completely loaded</a>.
+        </li>
+        <li>If there is already an <a data-lt=
+        "present an install prompt">install prompt being presented</a> or if
+        the <a>steps to install the web application</a> are currently being
+        executed, then abort this step.
+        </li>
+        <li>
+          <a>Queue a task</a> on the <a>application life-cycle task source</a>
+          to do the following:
+          <ol>
+            <li>Let <var>event</var> be a newly constructed
+            <a>BeforeInstallPromptEvent</a> named
```

Comment:
Globally: 
```suggestion
            {{BeforeInstallPromptEvent}} named
```

I lets ReSpec sprinkles the right things for linking.